core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -275,5 +280,16 @@ oppia.controller('CollectionPlayer', [
         $scope.generatePathParameters();
       }
     }, true);
+
+    $scope.scrollToLocation = function(id) {
+      $location.hash(id);
+      $anchorScroll();
+    };
+    // Touching anywhere outside the mobile preview should hide it.
```

Comment:
I might not be understanding it correctly, but it still seems to me like the logic of previewCardMobileIsShown is the opposite of the variable name. Maybe it should be previewCardMobileIsHidden, or the usage of true/false changed? The function seems to be to hide it, but sets ...IsShown to true.